dotcom-rendering/src/web/components/ContainerLayout.tsx

```diff
@@ -164,19 +167,39 @@ export const ContainerLayout = ({
 					borderType={centralBorder}
 					borderColour={borderColour || overrides?.border.container}
 					size={leftColSize}
+					verticalMargins={verticalMargins}
 				>
-					<>
-						<ContainerTitle
-							title={title}
-							fontColour={fontColour || overrides?.text.container}
-							description={description}
-							url={url}
-							containerPalette={containerPalette}
-							showDateHeader={showDateHeader}
-							editionId={editionId}
-						/>
-						{leftContent}
-					</>
+					<div
+						css={css`
+							display: flex;
+							height: 100%;
+							flex-direction: column;
+							justify-content: space-between;
+						`}
+					>
+						<div>
+							<ContainerTitle
+								title={title}
+								fontColour={
+									fontColour || overrides?.text.container
+								}
+								description={description}
+								url={url}
+								containerPalette={containerPalette}
+								showDateHeader={showDateHeader}
+								editionId={editionId}
+							/>
+							{leftContent}
+						</div>
+						{treats && (
```

Comment:
Worth adding a guard against `treat.length > 0` to not add an empty list?
(This might be tied into me not understanding the `undefined` case of `treats` in the model)

Reply:
I added a check to the Treats component itself